llvm/lib/Target/AMDGPU/SIModeRegister.cpp

```diff
@@ -178,18 +180,21 @@ Status SIModeRegister::getInstructionMode(MachineInstr &MI,
       unsigned Mode = MI.getOperand(2).getImm();
       MI.removeOperand(2);
       // Replacing the pseudo by a real instruction in place
```

Comment:
We can probably remove these comments. It's pretty obvious we do setDesc() here.

Reply:
removed